internal/handlers/health.go

```diff
@@ -1,7 +1,12 @@
 package handlers
 
-import "net/http"
+import (
+	"log"
+	"net/http"
+)
 
-func Health(w http.ResponseWriter, r *http.Request) {
-	w.Write([]byte("OK"))
+func Health(w http.ResponseWriter, _ *http.Request) {
```

Comment:
I would generally recommend just squashing this type of fix into the commit which introduced the handler for simplicity's sake